### PR TITLE
Remove unused functions, fix typos

### PR DESCRIPTION
**Pull request by cmccrea23**
I used the Inspect Code tool in phpStorm to find and remove unused functions. If entire files could be deleted, I did so. If the functions were wrapped in `script` tags, I removed the tags as well. I also corrected typos I found and made sure to check for the misspellings in the entire project.

### CODE REVIEW

File: interface/drugs/add_edit_drug.php [hunk 1]
```diff
@@ -100,32 +100,6 @@ function writeTemplateLine($selector, $dosage, $period, $quantity, $refills, $pr
...
-function get_related() {
- return document.forms[0].form_related_code.value.split(';');
-}
-
-// This is for callback by the find-code popup.
-// Deletes the specified codetype:code from the currently selected list.
-function del_related(s) {
- my_del_related(s, document.forms[0].form_related_code, false);
-}
-
```
Comment (sunsetsystems) [inline interface/drugs/add_edit_drug.php]: These functions are referenced. See the comments above each.

**Comment by sunsetsystems** [comment]
I wonder how phpStorm determines that a JavaScript function is unused? I'm not sure it's doing a very good job of that.

**Comment by danehrlich1** [comment]
Rod, I noticed the same thing. There are definitely functions in here that are being used.

Wish we could just limit it to PHP files, but they have JS in them as of now...

I’ll do some manual checking tonight

**Comment by bradymiller** [comment]
If possible, would not touch the files that end in .html and .js . That would make then more manageable.

**Comment by cmccrea23** [comment]
Ok, I'll replace the functions in .html and .js files, as well as the functions in the file mentioned above. Do you have other rules of thumb I could use to identify when functions are and are not being used?

**Comment by danehrlich1** [comment]
Claire, that is a good but tough to answer question. 

Brady and I spoke about this in-depth last night.

I have a laptop with PHPStorm that I am picking up this afternoon (needed to get it fixed). I will look at this more on the plane home tonight.
Here are a couple of things I know for now:

1. Great job fixing the misspellings, that should stay. We’ll to do do a database patch later to match it, but we can take care of that. 
2. Any PHP file in a “forms” folder probably shouldn’t be touched (other than misspellings). A lot of times these are called in JS or HTML files which PHPStorm has trouble recognizing.
3. Look over code comments I would say too in PHP files. Are there any pointless comments that can be deleted?


NOTE: I am working on throwing up about 10 new issues that are beginner to intermediate by next Wednesday. Since you did FCC you should look at the Javascript ones, since they will further your skills there. I will let you know when they are up.

**Comment by sunsetsystems** [comment]
I think any seemingly unused function is worth an eyebrow raise. Best to search the code for references to it and try to understand what happened. In theory when code is obsoleted it should be removed at that time.

**Comment by cmccrea23** [comment]
Thanks for the suggestions. I looked at the comments on the functions before deleting them, but I can't tell whether the comments are still relevant or not. I did leave in functions that had comments saying they were buggy, as I assumed they were actively being worked on. I'll keep an eye out for the Javascript issues as well :)

**Comment by cmccrea23** [comment]
When I searched for unused functions while ignoring the js and html files, I didn't come up with anything. I did find more typos, though. Do you want me to open a separate PR for these?

**Comment by danehrlich1** [comment]
Yes for the typos that would be great to get those fixed.

Make it a separate PR, because fixing some of those typos might require additional stuff from us on other stuff (e.g. issuing a patch for the database to reflect the changes you make). Will
be handy to have a separate PR we can comment on. 

I will work on creating other beginner / intermediate issues for you by end of the week as well that you can choose from based on what you find most interesting.

**Comment by cmccrea23** [comment]
Thank you!

**Pull request closed without merging**